draft-ietf-httpbis-http2bis.xml

```diff
@@ -262,21 +262,26 @@
     <section anchor="starting">
       <name>Starting HTTP/2</name>
       <t>
-        An HTTP/2 connection is an application-layer protocol running on top of a TCP connection
-        (<xref target="TCP"/>). The client is the TCP connection initiator.
+        Implementations that generate HTTP requests need to discover whether a server supports
+        HTTP/2.
       </t>
       <t>
-        HTTP/2 uses the "http" and "https" URI schemes defined in <xref target="HTTP" section="4.2"/>, with the same default port numbers.  As a result,
-        implementations processing requests for target resource URIs like <tt>http://example.org/foo</tt> or <tt>https://example.com/bar</tt> are required to first discover whether the
-        upstream server (the immediate peer to which the client wishes to establish a connection)
-        supports HTTP/2.
+        HTTP/2 uses the "http" and "https" URI schemes defined in <xref target="HTTP"
+        section="4.2"/>, with the same default port numbers as HTTP/1.1 (<xref
+        target="HTTP11"/>). These URIs do not include any indication about what versions an upstream
+        server (the immediate peer to which the client wishes to establish a connection) supports
+        HTTP/2.
       </t>
       <t>
         The means by which support for HTTP/2 is determined is different for "http" and "https"
         URIs.  Discovery for "https" URIs is described in <xref target="discover-https"/>. HTTP/2
         support for "http" URIs can only be discovered by out-of-band means, and requires prior knowledge
         of the support as described in <xref target="known-http"/>.
       </t>
+      <t>
+        An HTTP/2 connection is an application-layer protocol running on top of a TCP connection
+        (<xref target="TCP"/>). The client is the TCP connection initiator.
+      </t>
```

Comment:
I think moving this down is fine but this paragraph now reads a bit strangely. It feels like a dangling paragraph fragment, like we wanted to put this information in somewhere but couldn't find anywhere good so we jammed it here. Which I guess is accurate, but I'd like to be able to hide that.

We have defined the term "client" in the glossary above, so we don't need to redefine it here. If we want we can move the "TCP connection initiator" piece up to the glossary too. We could then move the "application-layer protocol running on top of a TCP connection" up to the overview. WDYT?